ts/src/program/index.ts

```diff
@@ -79,6 +79,7 @@ export class Program<IDL extends Idl = Idl> {
    * });
    * ```
    * @deprecated
+   * Use program.methods.<method>(...args) instead of the rpc call
```

Comment:
My suggestion was a generalized function call because initialize is just one of the function calls but if you feel this is more appropriate i can commit this

Reply:
> The generalized function call is good, but you need to add the `.rpc()` at the end.

Done!